src/lib/shaders/hit-test-shaders.ts

```diff
@@ -65,6 +88,7 @@ uniform float ts;
  * and height of the bounding box of interest. Currently those are ignored.
  */
 uniform vec4 hitTestCoordinates;
+uniform float inclusive;
```

Comment:
Why does this have to be a `float` instead of a `bool`?

Reply:
Good point. I did not realize `bool` was an option for uniforms. Fixing.